base/uk.ac.stfc.isis.ibex.ui.devicescreens/src/uk/ac/stfc/isis/ibex/ui/devicescreens/models/EditDeviceScreensDescriptionViewModel.java

```diff
@@ -101,73 +115,43 @@ public EditDeviceScreensDescriptionViewModel(DeviceScreensModel deviceScreensMod
         this.provider = provider;
 
         canWriteRemote = deviceScreensModel.isCanWriteRemote();
-        setPersistenceEnabled();
 
         // From the description create a list of devices
-        devices = new ArrayList<>();
+        ArrayList<DeviceDescriptionWrapper> screens = new ArrayList<>();
         for (DeviceDescription d : deviceScreensModel.getDeviceScreensDescription().getDevices()) {
-            devices.add(new DeviceDescriptionWrapper(d, this.provider));
+            screens.add(new DeviceDescriptionWrapper(d, this.provider));
```

Comment:
No, we can't switch :( Can't remember the exact reason off the top of my head, Matt looked into it.